R/minimize.R

```diff
@@ -101,12 +101,31 @@ minimize <- function(objective, subject_to, initial_design,
             cont_design <- update(cont_design, res2$solution)
             cont_design <- make_tunable(cont_design, n1, n2_pivots)
 
-            return(cont_design)
+            out <- list(
+                "design" = cont_design,
+                "nloptr_output" = res,
+                "nloptr_output_post_processing" = res2
+                )
 
+            if(res$iterations == opts$maxeval){
```

Comment:
this is not clean - nloptr already returns a status integer and messages. those should be used directly!